doc/howto/network_create.md

```diff
@@ -1,15 +1,77 @@
 # How to create and configure a network
 
-The desired type can be specified using the `--type` argument, e.g.
+To create and configure a managed network, use the `lxc network` command and its subcommands.
+Append `--help` to any command to see more information about its usage and available flags.
+
+(network-types)=
+## Network types
+
+The following network types are available:
+
+```{list-table}
+   :header-rows: 1
+* - Network type
+  - Documentation
+  - Configuration options
+* - `bridge`
+  - {ref}`network-bridge`
+  - {ref}`network-bridge-options`
+* - `ovn`
+  - {ref}`network-ovn`
+  - {ref}`network-ovn-options`
+* - `macvlan`
+  - {ref}`network-macvlan`
+  - {ref}`network-macvlan-options`
+* - `sriov`
+  - {ref}`network-sriov`
+  - {ref}`network-sriov-options`
+* - `physical`
+  - {ref}`network-physical`
+  - {ref}`network-physical-options`
+```
+
+## Create a network
+
+Use the following command to create a network:
 
 ```bash
-lxc network create <name> --type=bridge [options...]
+lxc network create <name> --type=<network_type> [configuration_options...]
 ```
 
-If no `--type` argument is specified, the default type of `bridge` is used.
+See {ref}`network-types` for a list of available network types and links to their configuration options.
+
+If you do not specify a `--type` argument, the default type of `bridge` is used.
+
+### Create a network in a cluster
+
+If you are running a LXD cluster and want to create a network, you must create the network for each cluster node separately.
```

Comment:
We shouldn't mention "node" in user facing documentation or errors. We say "cluster member" instead.